scripts/taxonomy/classification.py

```diff
@@ -41,7 +39,7 @@
 )
 @click_config_file.configuration_option()
 @click.pass_context
-def main(
+def generate_test_records(
```

Comment:
Replace the script to a code to generate test records for indexing purposes. It does the prediction on records that don't have the target label field, and writes those prediction into json.